pom.xml

```diff
@@ -313,10 +313,10 @@
     <version.org.jboss.arquillian.extension.drone>2.5.1</version.org.jboss.arquillian.extension.drone>
     <version.org.jboss.arquillian.graphene>2.3.2</version.org.jboss.arquillian.graphene>
     <version.com.google.guava.for-selenium>23.0</version.com.google.guava.for-selenium>
-    <version.com.google.elemental2>1.0.0-RC1</version.com.google.elemental2>
-    <version.org.jboss.errai>4.8.0.Final</version.org.jboss.errai>
+    <version.com.google.elemental2>1.1.0</version.com.google.elemental2>
+    <version.org.jboss.errai>4.9.0.Final</version.org.jboss.errai>
     <!-- WildFly version used together with  the GWT's Super Dev Mode -->
-    <version.org.jboss.errai.wildfly>19.1.0.SP1</version.org.jboss.errai.wildfly>
+    <version.org.jboss.errai.wildfly>19.1.0.Final</version.org.jboss.errai.wildfly>
```

Comment:
This is not correct. we should keep 19.1.0.SP1 version as the 19.1.0.Final wrapped Errai WF distro has wrong execute permission in zip file